libs/ui/src/components/NeoModalHead/NeoModalHead.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <header v-if="isMobile" class="modal-card-head">
     <div class="modal-card-title" @click.stop="emit('close')">
-      <NeoIcon icon="chevron-left" />
+      <NeoIcon icon="chevron-left" class="is-clickable" />
```

Comment:
**suggestion**: put click event on `button` or `a` tag not on icon

Reply:
fixed